http.go

```diff
@@ -377,6 +378,21 @@ func (rnr *httpRunner) Run(ctx context.Context, r *httpRequest) error {
 	d[httpStoreRawBodyKey] = string(resBody)
 	d[httpStoreHeaderKey] = res.Header
 
+	cookies := res.Cookies()
+
+	if len(cookies) > 0 {
+		keyMap := make(map[string]*http.Cookie)
+
+		for _, c := range cookies {
+			// If the Domain attribute is not specified, the host is taken over
+			c.Domain = rnr.endpoint.Host
+			keyMap[c.Name] = c
+		}
+
+		d[httpStoreCookieKey] = keyMap
```

Comment:
If there is no SetCookie in the response header, should I explicitly set an empty slice?

Reply:
I don't think empty slice is necessary because it doesn't become panic even if it is nil, but I thought it would be better to have empty map set.

https://github.com/k1LoW/runn/blob/867ca0cff63c8b5e7a2eb51e723664728cd18260/store.go#L131

https://github.com/k1LoW/runn/blob/867ca0cff63c8b5e7a2eb51e723664728cd18260/store.go#L155-L157

Reply:
Fixed this
https://github.com/k1LoW/runn/pull/556/commits/9d2c60c6104a55e1297a0011b9f2120a25ab83af